guides/common/modules/proc_disabling-dns-for-integration.adoc

```diff
@@ -0,0 +1,37 @@
+[id="disabling-dns-for-integration"]
+= Disabling DNS for integration
+
+If you want to manually manage a DNS service, you must prevent {Project} from maintaining this service on the operating system and disable orchestration to avoid errors.
+
+[NOTE]
+====
+Disabling DNS in {Project} does not remove the related backend service on the operating system.
+====
+
+.Prerequisites
+* You are logged in to the {ProjectWebUI}.
+
+.Procedure
+. Navigate to *Infrastructure* > *Subnets*.
```

Comment:
```suggestion
. In the {ProjectWebUI}, navigate to *Infrastructure* > *Subnets*.
```

Reply:
Is this necessary? The prereq already says that you need to be in the Web UI.